Backend/admin/src/screens/news/news.css

```diff
@@ -1,12 +1,11 @@
 .content{
     display: flex;
-    flex-wrap: wrap;
     flex: 1 1 auto;
-    justify-content: center;
+    justify-content: space-between;
     align-items: center;
 }
 
 .margin{
-    width: 200px;
+    width: 470px;
```

Comment:
Fix width ? Won t be better to have a min and a max ? Though the card is small enough.
But still the css class is named margin